crates/release_plz/src/args/release.rs

```diff
@@ -52,13 +55,25 @@ impl TryFrom<Release> for ReleaseRequest {
         } else {
             None
         };
+        let token = r.token.map(SecretString::from);
         Ok(ReleaseRequest {
             local_manifest: local_manifest(r.project_manifest.as_deref()),
             registry: r.registry,
-            token: r.token.map(SecretString::from),
+            token: token.clone(),
             dry_run: r.dry_run,
             git_release,
-            repo_url: r.repo_url,
+            repo_url: r.repo_url.clone(),
+            backend: match r.git_backend {
+                GitBackendKind::Github => GitBackend::Github(GitHub::new(
+                    RepoUrl::new(r.repo_url.clone().unwrap().as_str())?.name,
+                    RepoUrl::new(r.repo_url.unwrap().as_str())?.owner,
+                    token.unwrap(),
```

Comment:
this is the token used for the cargo registry. You need to use the `git_token` instead.